pvmismatch/pvmismatch_lib/pvmodule.py

```diff
@@ -194,7 +200,14 @@ def __init__(self, cell_pos=STD96, pvcells=None, pvconst=None,
                 if p.pvconst is not pvconst:
                     raise Exception('PVconstant must be the same for all cells')
         self.pvconst = pvconst  #: configuration constants
-        self.Vbypass = Vbypass  #: [V] trigger voltage of bypass diode
+        
+        # set default value of Vbypass if None
+        if not Vbypass:
+            self.Vbypass = VBYPASS  #: [V] trigger voltage of bypass diode
```

Comment:
You should change this to `if Vbypass is not None:` or else you will get the default constant `VBYPASS` if you try to set `Vbypass = 0` which also evaluates as `False`